include/picongpu/CMakeLists.txt

```diff
@@ -365,14 +382,14 @@ add_library(picongpu-hostonly
     STATIC
     ${SRCFILES}
 )
-target_link_libraries(picongpu-hostonly ${LIBS} m)
```

Comment:
it refers to the std math libraries.
On Windows, those are added by default (and adding them again leads to an error), but on all other OS one has to add that lib when accessing std math functions.